src/solvers/dgsem_tree/container_viscous_2d.jl

```diff
@@ -0,0 +1,61 @@
+mutable struct ViscousContainer2D{uEltype <: Real}
+    u_transformed::Array{uEltype, 4}
+    gradients::Vector{Array{uEltype, 4}}
+    flux_viscous::Vector{Array{uEltype, 4}}
+
+    # internal `resize!`able storage
+    _u_transformed::Vector{uEltype}
+    _gradients::Vector{Vector{uEltype}}
+    _flux_viscous::Vector{Vector{uEltype}}
```

Comment:
For this we need actually vectors, even for the outer dimensions such that the `resize!` operations work.

Reply:
Hm, I didn't expect that. So you cannot simply call `resize!(_gradients[1])` to resize individual tuple entries? Is this due to immutability of tuples?

Reply:
Hm hold on, that is actually fine - I think I confused myself while changing too many things at the same time.

Reply:
In that case, one could actually think about switching to `Tuple` since there is no additional code introduced by doing so, in contrast for the `outer` datastructures (see above).

Reply:
Sure. But if we need Vector types for the unsafe_wrap call, I am ok with just using Vector

Reply:
I pushed the versions where we use tuples for `_gradients` and `_flux_viscous`.

Reply:
Thanks!